src/main/resources/com/puppycrawl/tools/checkstyle/grammars/javadoc/JavadocParser.g4

```diff
@@ -878,7 +878,8 @@ javadocTag: AUTHOR_LITERAL (WS | NEWLINE)* description?
 
       | RETURN_LITERAL (WS | NEWLINE)* description?
 
-      | SEE_LITERAL (WS | NEWLINE)* reference? (STRING | htmlElement)* (WS | NEWLINE)* description?
+      | SEE_LITERAL (WS | NEWLINE)* reference? ((WS | NEWLINE)+ STRING | htmlElement)*
+      (WS | NEWLINE)* ((WS | NEWLINE)+ description)?
```

Comment:
@ps-sp ,

I also think that 

>  ((WS | NEWLINE)+ STRING | htmlElement)* (WS | NEWLINE)* ((WS | NEWLINE)+ description)

should be 
`((WS | NEWLINE)+ STRING | htmlElement)* ((WS | NEWLINE)+ description)`

your case `@see something` should match to my mind, as description should preceed with space ,"+" will demand it.